core/ui/src/main/res/values/strings.xml

```diff
@@ -97,6 +97,12 @@
     <!-- datetime layout -->
     <string name="event_time_label">Event time</string>
 
+    <!-- time/timezone/DST changes -->
```

Comment:
If string is not shared with other code, put it in values in module

Reply:
I thought of that. But I also noticed that these type of strings are very generic. We already have some degree of string duplication among drivers. I can still move this into the module, just wanted to bring this up to think about it.